maro/rl/training/__init__.py

```diff
@@ -1,18 +1,18 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+from .abs_algorithm import AbsAlgorithm, AlgorithmParams, MultiAlgorithm, SingleAlgorithm
```

Comment:
`MultiAlgorithm` a little bit strange = =
Actually it is a trainer for multi-agent algorithm?

Reply:
`SingleAlgorithm` => `SingleAgentAlgorithm`
`MultiAlgorithm` => `MultiAgentAlgorithm`